cypress/TestCases/FireboltCertification/Manage/ClosedCaptionsManage.feature

```diff
@@ -81,37 +81,37 @@ Feature: ClosedCaptions_Manage
+            | Set enabled-123             | enabled            | 123        | INVALID_PARAMS |
+            | Set fontFamily-123          | fontFamily         | 123        | INVALID_PARAMS |
+            | Set fontFamily-true         | fontFamily         | true       | INVALID_PARAMS |
+            | Set fontSize-true           | fontSize           | true       | INVALID_PARAMS |
+            | Set fontSize-test           | fontSize           | test       | INVALID_PARAMS |
+            | Set fontSize-true           | fontSize           | true       | INVALID_PARAMS |
+            | Set fontSize-0.25           | fontSize           | 0.25       | INVALID_PARAMS |
+            | Set fontFamily-sans-serif   | fontFamily         | sans-serif | INVALID_PARAMS |
+            | Set fontEdge-123            | fontEdge           | 123        | INVALID_PARAMS |
+            | Set fontEdge-true           | fontEdge           | true       | INVALID_PARAMS |
+            | Set fontEdge-solid          | fontEdge           | solid      | INVALID_PARAMS |
+            | Set preferredLanguages-true | preferredLanguages | true       | INVALID_PARAMS |
+            | Set preferredLanguages-123  | preferredLanguages | 123        | INVALID_PARAMS |
+            | Set preferredLanguages-test | preferredLanguages | test       | INVALID_PARAMS |
+            | Set windowColor-123         | windowColor        | 123        | INVALID_PARAMS |
+            | Set windowOpacity-true      | windowOpacity      | true       | INVALID_PARAMS |
+            | Set fontEdgeColor-123       | fontEdgeColor      | 123        | INVALID_PARAMS |
+            | Set fontEdgeColor-true      | fontEdgeColor      | true       | INVALID_PARAMS |
+            | Set backgroundColor-123     | backgroundColor    | 123        | INVALID_PARAMS |
+            | Set backgroundColor-true    | backgroundColor    | true       | INVALID_PARAMS |
+            | Set textAlign-123           | textAlign          | 123        | INVALID_PARAMS |
+            | Set textAlign-true          | textAlign          | true       | INVALID_PARAMS |
+            | Set textAlignVertical-123   | textAlignVertical  | 123        | INVALID_PARAMS |
+            | Set textAlignVertical-true  | textAlignVertical  | true       | INVALID_PARAMS |
+            | Set fontColor-123           | fontColor          | 123        | INVALID_PARAMS |
+            | Set fontColor-true          | fontColor          | true       | INVALID_PARAMS |
+            | Set fontOpacity-120         | fontOpacity        | 120        | INVALID_PARAMS |
+            | Set fontOpacity-test        | fontOpacity        | test       | INVALID_PARAMS |
+            | Set fontOpacity-true        | fontOpacity        | true       | INVALID_PARAMS |
+            | Set backgroundOpacity-120   | backgroundOpacity  | 120        | INVALID_PARAMS |
+            | Set backgroundOpacity-test  | backgroundOpacity  | test       | INVALID_PARAMS |
```

Comment:
if all of the values are same, why do we need a separate column here

Reply:
review comment addresed